### PR TITLE
FACE HTML file - Support Template Selection - Dropdown Menu

### PR DESCRIPTION
**Pull request by prajapac**
## Relevant Rally Links

https://rally1.rallydev.com/#/289692574792d/custom/355050439968?detail=%2Fuserstory%2F424774442084

## Description

As part of the new Feature work for HTML FACE pages, we would like to support HTML File Template selection. The first piece to this is to fetch the relevant Siren Entities for this, and and display the list of templates in a dropdown menu. (See Rally link for link to designs).

## Goal/Solution

This PR defines the ContentHtmlFileTemplatesEntity class that allows for easier retrieval of HTML File Templates.

## Video/Screenshots

Demo

## Related PRs

- https://github.com/BrightspaceHypermediaComponents/activities/pull/1849

## Remaining Work

- [ ] Dev Testing: another developer will test this code on their machine

### CODE REVIEW

**Review by quinnbudan — APPROVED** [review]
LGTM!

File: src/activities/content/ContentFileEntity.js [hunk 1]
```diff
@@ -141,4 +141,11 @@ export class ContentFileEntity extends ContentWorkingCopyEntity {
 		}
 		return null;
 	}
+
+	/**
+	 * @returns {string|null} html templates link
+	 */
+	getHtmlTemplatesHref() {
```
Comment (brissons) [inline src/activities/content/ContentFileEntity.js:148]: I wonder if this should be in the `ContentHtmlFileEntity` entity instead? Just to keep the more generic file stuff and HTML specific stuff separate.

**Review by ChrisKraljevicD2L — APPROVED** [review]

**Review by brissons — APPROVED** [review]
lgtm! with a small comment/suggestion.

File: src/activities/content/ContentFileEntity.js (hunk 1 above)
Comment (prajapac) [reply src/activities/content/ContentFileEntity.js:148]: I see how you mean, however I'm not sure it's worth blocking the current feature. I've made a note of this and see if I can do it in a follow-up/clean-up. Thanks!

**Pull request merged**